src/algorithms/bubbleSort/BubbleSort.java

```diff
@@ -0,0 +1,22 @@
+package src.algorithms.bubbleSort;
+
+public class BubbleSort {
+    /**
+     * Sorts the given array.
+     * @param arr array to be sorted.
+     * @return the same array arr that is sorted.
+     */
+    public static int[] sort(int[] arr) {
+        int n = arr.length;
+        for (int i = 0; i < n - 1; i++ ) {
+            for (int j = 0; j < n - 1 - i; j++) {
+                if (arr[j] > arr[j + 1]) {
```

Comment:
or in another example, short in-line comment to explain how changing to '<' will result in descending order (or rather, non-increasing if you wanna be precise)